crowbar_framework/app/models/ceph_service.rb

```diff
@@ -49,7 +49,8 @@ def role_constraints
           "unique" => false,
           "count" => 1,
           "platform" => {
-            "suse" => "12.0"
+            "suse" => "12.0",
+            "suse" => "12.1"
```

Comment:
If you want to support multiple versions you are forced to use regular expressions like `/12.[01]{1}/`